docs/generate_submission_key.rst

```diff
@@ -56,36 +56,36 @@ Create the Key
 Export the *Submission Public Key*
 ----------------------------------
 
-Navigate to **Applications ▸ Utilities ▸ Passwords and Keys** to open a
-graphical interface to manage GPG keys. Click the clipboard icon |gpgApplet|
-in the top right corner and select "Manage Keys". Click "GnuPG keys" and you
-should see the key that you just generated.
+Navigate to **Applications ▸ Utilities ▸ Kleopatra** to open a
```

Comment:
This is under "Accessories" for me at least on Tails 5.2.

Reply:
I double-checked, and you are correct about it not being in Utilities any longer. Apparently it can be found in both "Accessories" as well as "Favorites," which is shown by default when you open the Applications menu. Since the Favorites list can change, I'll specify "Accessories" here since that is consistently where it can be found.